packages/clerk-js/src/ui/elements/Avatar.tsx

```diff
@@ -13,7 +12,6 @@ type AvatarProps = PropsOfComponent<typeof Flex> & {
   initials?: string;
   imageUrl?: string | null;
   imageFetchSize?: number;
-  optimize?: boolean;
```

Comment:
❓ isn't this a breaking change? Couldn't someone use our components (eg `OrganizationAvatar`) with the `optimize` prop?

Reply:
This is just an internal prop, not really needed right now as the new worker and HOC handles this automatically